src/core/proj/qgscoordinatetransform.cpp

```diff
@@ -829,6 +829,9 @@ void QgsCoordinateTransform::transformCoords( int numPoints, double *x, double *
     }
   }
 
+  // something which won't clash with Proj's real error codes!
+  constexpr int PROJ_RESULT_FALLBACK_OPERATION_FAILED = -481516;
```

Comment:
How can we be sure that it won't clash in next releases? Wouldn't be better to add an extra bool ?

Reply:
@troopa81 ok, done